filters/resizebywidth_test.go

```diff
@@ -0,0 +1,41 @@
+package filters
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/zalando-incubator/skrop/filters/imagefiltertest"
+	"testing"
+)
+
+func TestNewResizeByWidth(t *testing.T) {
+	name := NewResizeByWidth().Name()
+	assert.Equal(t, "width", name)
+}
+
+func TestResizeByWidth_Name(t *testing.T) {
+	c := resizeByWidth{}
+	assert.Equal(t, "width", c.Name())
+}
+
+func TestResizeByWidth_CreateOptions(t *testing.T) {
+	resizeByWidth := resizeByWidth{width: 256}
+	image := imagefiltertest.LandscapeImage()
+	options, _ := resizeByWidth.CreateOptions(image)
+
+	assert.Equal(t, 256, options.Width)
+}
+
+func TestResizeByWidth_CreateFilter(t *testing.T) {
+	imagefiltertest.TestCreate(t, NewResizeByWidth, []imagefiltertest.CreateTestItem{{
+		Msg:  "no args",
+		Args: nil,
+		Err:  true,
+	}, {
```

Comment:
you should have tests to test the image size conditions. If the image's withs is bigger than the requested width, etc